geoportailv3/static/less/offline.less

```diff
@@ -0,0 +1,58 @@
+ngeo-offline {
```

Comment:
Have you considered using the less file from ngeo instead of duplicating it?

Reply:
It's not ngeo core css, it's the css from the example, they are not exported in ngeo npm package.